### ISSUE TITLE
National Historic Trail auto tour routes

### ISSUE THREAD

**Issue by quincylvania** [ZeLonewolf/openstreetmap-americana#1101]
There are 21 [National Historic Trails](https://en.wikipedia.org/wiki/National_Trails_System#National_Historic_Trails) as of 2024. Unlike other members of the National Trails System, the historic trails tend to be signed as auto tour routes on modern roads rather than hiking trails. These routes are _very_ Americana and the shields would make a great addition to the style.

- [x] ~~[Ala Kahakai](https://en.wikipedia.org/wiki/Ala_Kahakai_National_Historic_Trail)~~ (offroad trail)
- [ ] [Butterfield Overland](https://en.wikipedia.org/wiki/Butterfield_Overland_National_Historic_Trail)
- [ ] [California](https://en.wikipedia.org/wiki/California_Trail)
  - [ ] Mapped
  - [ ] Shield
- [x] ~~[Captain John Smith Chesapeake](https://en.wikipedia.org/wiki/Captain_John_Smith_Chesapeake_National_Historic_Trail)~~ (water trail)
- [x] ~~[Chilkoot](https://en.wikipedia.org/wiki/Chilkoot_Trail)~~ (offroad trail)
- [ ] [El Camino Real de los Tejas](https://en.wikipedia.org/wiki/El_Camino_Real_de_los_Tejas_National_Historic_Trail)
- [ ] [El Camino Real de Tierra Adentro](https://en.wikipedia.org/wiki/El_Camino_Real_de_Tierra_Adentro_National_Historic_Trail)
- [x] ~~[Iditarod](https://en.wikipedia.org/wiki/Iditarod_National_Historic_Trail)~~ (offroad trail)
- [ ] [Juan Bautista de Anza](https://en.wikipedia.org/wiki/Juan_Bautista_de_Anza_National_Historic_Trail)
  - [ ] Mapped
  - [ ] Shield
- [ ] [Lewis and Clark](https://en.wikipedia.org/wiki/Lewis_and_Clark_National_Historic_Trail)
  - [ ] Mapped
  - [ ] Shield
- [ ] [Mormon Pioneer](https://en.wikipedia.org/wiki/Mormon_Pioneer_National_Historic_Trail)
  - [ ] Mapped
  - [ ] Shield
- [ ] [Nez Perce (Nee-Me-Poo)](https://en.wikipedia.org/wiki/Nez_Perce_National_Historic_Trail)
  - [ ] Mapped
  - [ ] Shield
- [ ] [Old Spanish](https://en.wikipedia.org/wiki/Old_Spanish_National_Historic_Trail)
  - [ ] Mapped
  - [ ] Shield
- [ ] [Oregon](https://en.wikipedia.org/wiki/Oregon_National_Historic_Trail)
  - [ ] Mapped
  - [ ] Shield
- [ ] [Overmountain Victory](https://en.wikipedia.org/wiki/Overmountain_Victory_National_Historic_Trail)
  - [ ] Mapped
  - [ ] Shield
- [ ] [Pony Express](https://en.wikipedia.org/wiki/Pony_Express_National_Historic_Trail)
  - [ ] Mapped
  - [ ] Shield
- [ ] [Santa Fe](https://en.wikipedia.org/wiki/Santa_Fe_National_Historic_Trail)
  - [ ] Mapped
  - [ ] Shield
- [ ] [Selma to Montgomery](https://en.wikipedia.org/wiki/Selma_to_Montgomery_National_Historic_Trail)
  - [ ] Mapped
  - [ ] Shield
- [ ] [Star-Spangled Banner](https://en.wikipedia.org/wiki/Star-Spangled_Banner_National_Historic_Trail)
  - [ ] Mapped
  - [ ] Shield
- [ ] [Trail of Tears](https://en.wikipedia.org/wiki/Trail_of_Tears_National_Historic_Trail)
  - [ ] Mapped
  - [ ] Shield
- [ ] [Washington–Rochambeau Revolutionary Route](https://en.wikipedia.org/wiki/Washington%E2%80%93Rochambeau_Revolutionary_Route)
  - [ ] Mapped
  - [ ] Shield

**Comment by ZeLonewolf** [comment]
This is great!  The Washington-Rochambeau route is of particular interest to me because it goes through my area.  I've already mapped the section from the Connecticut border to the end in Newport.  There are signs all along the route!

Here's the route relation so far: https://www.openstreetmap.org/relation/15649521
I assigned it a network value of `US:W3R` based on the [organization](https://w3r-us.org/) that manages the route.

**Comment by claysmalley** [comment]
> I assigned it a network value of `US:W3R` based on the [organization](https://w3r-us.org/) that manages the route.

The [Selma to Montgomery NHT](https://www.openstreetmap.org/relation/13302056) is currently mapped with `network=US:NHT`. Would it be better to continue using this network value, and distinguish routes by name, as is done with several other touristy route networks? Do these routes have similar shields on the ground?

**Comment by ZeLonewolf** [comment]
The Washington-Rochambeau route doesn't even have a consistent sign along its route.
This one is used in RI:
![image](https://github.com/ZeLonewolf/openstreetmap-americana/assets/3254090/051cac3c-993a-484b-a39c-ac25a9202670)
But this one also seems to be in use in other states:
![image](https://github.com/ZeLonewolf/openstreetmap-americana/assets/3254090/e345bc46-d36d-48ca-8702-534d35755f78)

**Comment by ZeLonewolf** [comment]
> > I assigned it a network value of `US:W3R` based on the [organization](https://w3r-us.org/) that manages the route.
> 
> The [Selma to Montgomery NHT](https://www.openstreetmap.org/relation/13302056) is currently mapped with `network=US:NHT`. Would it be better to continue using this network value, and distinguish routes by name, as is done with several other touristy route networks?

I agree this is a more correct tagging scheme given that these trails are all part of a network.  I don't know what to do about the competing shield situation though.

**Comment by quincylvania** [comment]
I think using `route=road` + `network=US:NHT` is okay as long as people know these refer to the auto routes and not the designated historic trails themselves (which may or may not exist as modern trails). Not all 21 trails have auto routes.

As for the shields, all the trails have standard triangle-ish markers that seem to be the modern preference for signage. I think the competing signs are older, either predating the NHT designation or the introduction of the standard signage.

![Generic_Family_of_Road_Signs_2020highres](https://github.com/ZeLonewolf/openstreetmap-americana/assets/2046746/dfe701be-4ddf-4dfa-affe-4767370f9bb3)

**Comment by 1ec5** [comment]
There’s also an “Auto Tour Route” sign as of the [MUTCD 11th edition](https://mutcd.fhwa.dot.gov/pdfs/11th_Edition/Chapter2d.pdf#page=73):

![Examples of Directional and Confirming assemblies for National Historic Trails](https://github.com/ZeLonewolf/openstreetmap-americana/assets/1231218/1ecd6a55-da7b-4125-b0f4-6691d621a4a8)

**Comment by 1ec5** [comment]
Like the Washington–Rochambeau, the Lewis and Clark Trail is signposted and marked on maps using a distinct shield:

[<img src="https://upload.wikimedia.org/wikipedia/commons/0/00/Lewis_and_Clark_Trail.svg" width="250" alt="Lewis and Clark Trail">](https://commons.wikimedia.org/wiki/File:Lewis_and_Clark_Trail.svg)

<img width="404" alt="MoDOT 2009" src="https://github.com/ZeLonewolf/openstreetmap-americana/assets/1231218/b9cc6b50-d80c-4f74-9b0a-40f04443c061">

**Comment by 1ec5** [comment]
<img width="440" alt="Iowa DOT 2010" src="https://github.com/ZeLonewolf/openstreetmap-americana/assets/1231218/b7227763-89cc-42cd-aa50-034800fabb90">

**Comment by quincylvania** [comment]
I've been mapping some of these, and I'm wondering if it's an issue to have super routes with subrelations for each state or section, with names like `name=Oregon National Historic Trail Auto Tour Route (KS)`. All components of the same route presumably have the same shield, and no `ref` value, so we'd want to simply use `overrideByName` like so:

```
overrideByName: {
      "Oregon National Historic Trail Auto Tour Route": {
        spriteBlank: "shield_us_nht_oregon",
      },
```

Can we do this without having a separate entry for each name (i.e. ignore the trailing parens)? Or ingest the super route name? I don't really want to need to have ten subrelations with identical names, nor squish together all the component routes.

**Comment by ZeLonewolf** [comment]
I think that the "(State)" piece should not be in the `name` tag because it's not part of the name.  That can go in `description` and also solves the technical issue.

**Comment by quincylvania** [comment]
> I think that the "(State)" piece should not be in the name tag because it's not part of the name.

Okay I see that seems to be the convention, thanks.

**Comment by 1ec5** [comment]
> `is_in:state` might also be an option here.

Yes, this is the structured option, with `description` as a convenience to mappers.

Note that OpenMapTiles only considers the tags on the relations that directly contain a given way, not any of their `route` superrelations or `superroute` superrelations.

**Comment by quincylvania** [comment]
I've updated the original post to link to federal restrictions on the trail insignia, if found.

**Comment by ZeLonewolf** [comment]
Well, these signs just started showing up in RI!

![20240902_141937](https://github.com/user-attachments/assets/3d74d4f6-7e1f-4adf-8ceb-cd59846a8ae4)

**Comment by Molandfreak** [comment]
Anyone to add a couple of shields?
Trail of Tears has been mapped: https://www.openstreetmap.org/relation/20462730
Nez Perce Trail has been mapped: https://www.openstreetmap.org/relation/20509039

**Comment by quincylvania** [comment]
Thanks so much for mapping these, @Molandfreak! I know from experience how much work this can be.

I'm wondering what your data source was for the Trail of Tears tour, or if it's just based on signage? Specifically, I'm looking at [this older relation](https://www.openstreetmap.org/relation/14335243) and seeing that it doesn't overlap.
